src/pages/watch/watch.js

```diff
@@ -0,0 +1,67 @@
+import styles from "./watch.module.scss"
+import { VideoEmbed } from "@components"
+import { Avatar } from "react-felix-ui"
+import { useVideos } from "@providers/video-provider"
+import { useWatchLater } from "@providers/watch-later-provider"
```

Comment:
question - can you tell me a bit about `@providers`. Have you published these like your cl ?

Reply:
I have set allies for my folder path with `react-app-rewire-alias` package